Source/WebCore/platform/cocoa/WebAVPlayerLayer.mm

```diff
@@ -106,32 +105,33 @@ - (void)dealloc
 {
     [NSObject cancelPreviousPerformRequestsWithTarget:self selector:@selector(resolveBounds) object:nil];
     [_pixelBufferAttributes release];
-    if (_fullscreenModel)
-        _fullscreenModel->removeClient(*_fullscreenModelClient);
+    if (auto model = _fullscreenModel.get())
+        model->removeClient(*_fullscreenModelClient);
```

Comment:
_fullscreenModelClient could become a UniqueRef.

Reply:
it's obj-C, there's no initialiser that would be able to set it up